### PR TITLE
Add elasticsearch docker image

### PR DESCRIPTION
**Pull request by qzheng527**
Signed-off-by: Qi Zheng <qi.zheng@intel.com>

### CODE REVIEW

File: elasticsearch/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	done
+
+COPY config /etc/elasticsearch
+COPY docker-entrypoint.sh /
+
+VOLUME /var/data/elasticsearch
+
+EXPOSE 9200 9300
+ENTRYPOINT ["/docker-entrypoint.sh"]
+CMD ["-Epath.conf=/etc/elasticsearch/"]
```
Comment (gtkramer) [inline elasticsearch/Dockerfile]: Can you double check the command here? I would expect something like the [upstream CMD](https://github.com/docker-library/elasticsearch/blob/master/Dockerfile-debian.template#L66).

**Comment by gtkramer** [comment]
Other than that one thing, everything else looks good.  Thanks for the help here, @bryteise with the previous PR.

File: elasticsearch/Dockerfile (hunk 1 above)
Comment (qzheng527) [reply elasticsearch/Dockerfile]: I use this command "-Epath.conf=/etc/elasticsearch/" to set the default configuration path in /etc/ to respect the stateless design of Clear Linux.

Comment (qzheng527) [reply elasticsearch/Dockerfile]: Using the upstream CMD implies the configuration path is in /usr/share/elasticsearch

**Pull request merged**